app/src/main/java/com/sdp/movemeet/Activity/ActivityDescriptionActivity.java

```diff
@@ -159,12 +158,17 @@ public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
                 FirebaseInteraction.logoutIfUserNonNull(fAuth, this);
                 break;
             case R.id.nav_start_activity:
+                Navigation.startActivity(this.navigationView);
```

Comment:
Why did you add this line?

Reply:
I think it's because of the code style update